src/main/app/src/Components/Landscape/Modals/Group/Group.tsx

```diff
@@ -1,87 +1,104 @@
-import React, { useEffect, useState } from 'react';
-import { get } from '../../../../utils/API/APIClient';
-import './Group.scss';
+import React from 'react';
 
-import { IAssessmentProps, IGroup } from '../../../../interfaces';
-import { getLabels, getLinks, getGroupItems, getAssessmentSummary } from '../../Utils/utils';
+import { IAssessment, IGroup } from '../../../../interfaces';
+import { getLabels, getLinks, getAssessmentSummary, getItemIcon } from '../../Utils/utils';
+import { Card, CardHeader } from '@material-ui/core';
+import CardContent from '@material-ui/core/CardContent';
+import StatusChip from '../../../StatusChip/StatusChip';
+import componentStyles from '../../../../Ressources/styling/ComponentStyles';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import Avatar from '@material-ui/core/Avatar';
 
 interface Props {
-  fullyQualifiedGroupIdentifier: string;
+  group: IGroup;
+  assessments: IAssessment;
   findItem?: (fullyQualifiedGroupIdentifier: string) => void;
   findGroup?: (fullyQualifiedGroupIdentifier: string) => void;
-  onAssessmentClick?: (fullyQualifiedItemIdentifier: string) => void;
 }
 
 /**
  * Returns a choosen Landscape group if informations are available
```

Comment:
Here it is also "chosen" and not "choosen"

Reply:
fixed